spec/features/01_visitor_can_add_bourbon_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
```

Comment:
With feature tests, please feel free to just name them semantically and without numbers in the name: i.e. "visitor_can_add_bourbon_spec.rb". 

I think some of the confusion around this comes from the fact that we sometimes name our tests numerically in your challenges or system checks in order to enforce an order that they run in and therefore a challenge design that helps you complete a task. In a professional setting, you'll want your tests to all be able to pass independently of one another and randomized by Rspec. A semantic name that helps someone looking for the test later is all that is needed.